src/main/include/subsystems/Climber.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <rev/CANSparkMax.h>
+#include <frc/DigitalInput.h>
+
+class Climber
+{
+    public:
+
+    Climber();
+
+    void move(double inches_from_base);
+
+    void init();
+
+    void periodic();
+
+    bool setLock(bool state);
+
+    bool getIsLocked();
```

Comment:
Instead of talking in terms of getters and setters (which is something of what is known as a "leaky abstraction"), why not just talk in English? Suggest making your action functions verbs and your query functions interrogative phrases:

```suggestion
    void Lock();
    void Unlock();
    bool IsLocked();
```